ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.ts

```diff
@@ -76,65 +76,18 @@ export class TissueComponent {
       }
     }
     if (v !== null) {
+      const convertedColumnName = DynamicValueUtilModel.convertToCamelCase(colName);
       if (this.tissue.additionalValuesJson != null) {
-        this.tissue.additionalValuesJson[colName] = v;
+        this.tissue.additionalValuesJson[convertedColumnName] = v;
       } else {
         const addArray = {};
-        addArray[colName] = v;
+        addArray[convertedColumnName] = v;
         this.tissue.additionalValuesJson = addArray;
       }
       this.valueChanged(this.tissue.additionalValuesJson, 'additionalValuesJson');
     }
   }
 
-  // display additional value
-  getAdditionalValue(colName: string): string {
-    if (this.tissue.additionalValuesJson != null) {
-      if (this.tissue.additionalValuesJson[colName] != null) {
-        return this.tissue.additionalValuesJson[colName];
-      } else if (this.tissue.additionalValuesJson[this.convertToCamelCase(colName)] != null) {
-        return this.tissue.additionalValuesJson[this.convertToCamelCase(colName)];
-      }
-    }
-    return null;
-  }
-
-  convertToCamelCase(str: string): string {
```

Comment:
This has moved to `dynamic-value-util.model.ts`, right? 
 Any changes, or just a move?

Reply:
Just a move since it was being referenced in both component and model and html files, and I suspect in future we will use that util class in other places like maybe onc history and mr fieldsettings as well (I did not do a full investigation around if the same problem is happening in other places yet, but given the way backend is formed, it might)